docs/stable/1-client.md

```diff
@@ -232,7 +241,7 @@ See [Appendix A](#appendix-a-security-considerations)
 
 ## Design Rationale
 
-### P2P Overlay
+### P2P Overlay 1
```

Comment:
Extra number?

Reply:
These things are my IDE. It forces unique anchors. I'll change it.